test/View/Console/DefaultRenderingStrategyTest.php

```diff
@@ -18,10 +18,13 @@
 use Zend\ServiceManager\ServiceManager;
 use Zend\Stdlib\Response;
 use Zend\View\Model;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class DefaultRenderingStrategyTest extends TestCase
 {
-    /** @var DefaultRenderingStrategy */
+    use EventManagerIntrospectionTrait;
+
+    /* @var DefaultRenderingStrategy */
```

Comment:
@weierophinney `/**` => docblock (stored in reflection data as well), while `/*` is only for IDE hints.

Here's the practical example: https://3v4l.org/O3CgU

And copying it for reference:

```php
<?php

class Foo
{
    /* stuff */
    private $bar;
    /** more stuff */
    private $baz;
}

var_dump((new ReflectionProperty(Foo::class, 'bar'))->getDocComment());
var_dump((new ReflectionProperty(Foo::class, 'baz'))->getDocComment());
```